exporters/zipkin/src/test/java/io/opentelemetry/exporter/zipkin/ZipkinSpanExporterEndToEndHttpTest.java

```diff
@@ -49,7 +49,7 @@
   private static final long SENT_TIMESTAMP_NANOS = 1505855799_459486280L;
   private static final Attributes attributes = Attributes.empty();
   private static final List<EventData> annotations =
-      ImmutableList.of(
+      Arrays.asList(
```

Comment:
Wrap into Collections.unmodifieableList?

Reply:
Since only used once went ahead and inlined